src/App.js

```diff
@@ -1,17 +1,45 @@
 import React, { Component } from 'react'
-
+import axios from 'axios';
 export default class App extends Component {
 
 constructor(props) {
   super(props);
   this.state = {
-    queryCity: ''
+    queryCity: '',
+    locationObject: {},
+    mapObject: {},
+    error: false
+  }
+}
+
+
+getLocation = async() => {
+  try {
+    let result = await axios.get(`https://us1.locationiq.com/v1/search.php?key=pk.7ce20acb246bf3b79f66fcb7d5ae9d3e&q=${this.state.queryCity}&format=json`);
+      // console.log(result);
+      this.setState({ locationObject: result.data[0]}, this.getMap);
+  } catch (error) {
+    console.error(error);
+    console.log('there was an error');
+    this.setState({ error: true })
+  }
+}
+getMap = async() => {
+  try {
+    console.log(this.state.locationObject.lat)
+    let result = await axios.get(`https://maps.locationiq.com/v3/staticmap?key=pk.7ce20acb246bf3b79f66fcb7d5ae9d3e&center=${this.state.locationObject.lat},${this.state.locationObject.lon}&zoom=1`);
```

Comment:
Good work on the lab so far!

When I go to the deployed site, I am seeing that the `mapObject` is a bunch of encoded characters. Ensure that all your API keys are added to `.env` and used as `process.env.REACT_API_KEY` inside url (see key here).

Reply:
To make your map display you don't actually need to make a get request, you can simply input the URL into the <img> tags src attribute. I would suggest setting the URL to a variable first, so you can do something like this: 

<img src="imageURL">

Check the "Static Map" Section of the docs page on the locationIQ website for more details.

Reply:
Is the url correct here?